src/cheerio_crawler.js

```diff
@@ -189,6 +194,7 @@ class CheerioCrawler {
 
         checkParamOrThrow(handlePageFunction, 'options.handlePageFunction', 'Function');
         checkParamOrThrow(requestFunction, 'options.requestFunction', 'Maybe Function');
+        if (requestFunction) log.warning('CheerioCrawler: options.requestFunction is deprecated. Use BasicCrawler instead.');
```

Comment:
I'd rather TODO to remove this when 1.0 is released  :)

Reply:
Do you mean to add a TODO or replace the deprecation warning by a TODO?